.ds_python_version

```diff
@@ -0,0 +1 @@
+7.0.0
```

Comment:
centralized the datascience python base image version to this file here.
this version makes its way into the dockerfile through all of our automated systems that build the image through push, release, and merge to master